.github/workflows/dotnet.yml

```
@@ -37,8 +35,8 @@ jobs:
     - name: Setup .Net
       uses: actions/setup-dotnet@v1
```

```suggestion
      uses: actions/setup-dotnet@v3
```
also update the other ones to `uses: actions/checkout@v2`